### PR TITLE
chore: change landing page for motoko docs

### PR DESCRIPTION
**Pull request by dfx-json**
Changes the landing page for the Motoko docs to the "Motoko Introduction".

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -172,7 +172,6 @@ const sidebars = {
       },
       items: [
         "motoko/version",
```
Comment (crusso) [inline sidebars.js:174]:
```suggestion
        "motoko/version",
        "motoko/main/motoko",
```
I wonder if it makes sense to also have a duplicate link here, just as an intro to the following docs. I'll leave it up to you. You can just commit the suggestion if you want to change, or leave it if not.

**Review by crusso — COMMENTED** [review]
LGTM. See minor suggestion you can take or leave.

BTW Develop -> Motoko Docs from the "World Computer is our future" initial page still takes you the first tutorial page by Mikhail rather than the landing page. Not sure if that is intended or not. I can live with it.

("World Computer is our future" makes me want to hurl, btw. Is it just me?)

File: sidebars.js (hunk 1 above)
Comment (dfx-json) [reply sidebars.js:174]: I agree in principal, but doing so causes two items to be highlighted at the same time:
Screenshot 2023-03-07 at 12 22 04 PM

**Review by crusso — APPROVED** [review]
Thanks for fixing those links!

**Review by olaszakos — APPROVED** [review]
LGTM

**Pull request merged**